webrtc.html

```diff
@@ -1780,6 +1785,13 @@ <h4>Set the RTCSessionDescription</h4>
                             <p>Set <var>transceiver</var>.<a>[[\FiredDirection]]</a> to
                             <var>direction</var>.</p>
                           </li>
+                          <li>
+                            <p>Set
+                              <var>transceiver</var>.<a>[[\Receiver]]</a>.<a>[[\ReceiveCodecs]]</a>
+                              to the codecs that <var>description</var>
+                              negotiates for receiving, and which the user
+                              agent is currently prepared to receive.</p>
```

Comment:
Optional: Same note here or is that superfluous?

Reply:
I think it's superfluous. I don't like repetition.